config/_config.space.scss

```diff
@@ -1,5 +1,5 @@
-$space: 16px; // Default AP space variable
+$space: 16px !default; // Default AP space variable
 
 // Bulma spacing
 
-$column-gap: $space/2;
+$column-gap: $space * 0.5
```

Comment:
As above, Using a divide is deprecated, so changed to multiply by 0.5 which returns the same result: 8px